features/step_definitions/metadata.js

```diff
@@ -16,13 +16,48 @@ Before(
         this.repoName = 'functional-metadata';
         this.pipelineId = null;
         this.eventId = null;
+        this.previousEventId = null;
         this.meta = null;
+        this.buildMeta = null;
         this.jwt = null;
     }
 );
 
 Given(/^a metadata starts with an empty object$/, { timeout: TIMEOUT }, () => null);
 
+Then(/^the "BOOZ" job is disabled$/, { timeout: TIMEOUT }, function step() {
+    return request({
+        url: `${this.instance}/${this.namespace}/pipelines/${this.pipelineId}/jobs?jobName=fourth`,
```

Comment:
can we avoid hardcoding `fourth`?

maybe set `this.jobName = 'fourth'` or another var?